app/api/api.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config();
+var express = require('express');
+var app = express();
+
+var db = require('../schemas');
+var config = require('../config');
+
+app.all('/api/*', (req, res, next) => {
+    res.set('Content-Type', 'application/json');
```

Comment:
Instead of doing this, you could just do `res.json(Object)` which automatically sets the `application/json` header and sends the object.